---a/net/minecraft/world/gen/structure/StructureMineshaftStart.java
+++b/net/minecraft/world/gen/structure/StructureMineshaftStart.java
@@ -1,14 +1,17 @@
 package net.minecraft.world.gen.structure;
 
 import java.util.Random;
+
 import net.minecraft.world.World;
 
-public class StructureMineshaftStart extends StructureStart
-{
+public class StructureMineshaftStart extends StructureStart {
+
     private static final String __OBFID = "CL_00000450";
 
-    public StructureMineshaftStart() {}
+    public StructureMineshaftStart() {
+    }
 
-    public StructureMineshaftStart(World worldIn, Random p_i2039_2_, int p_i2039_3_, int p_i2039_4_)
-    {
+    public StructureMineshaftStart(World worldIn, Random p_i2039_2_, int p_i2039_3_, int p_i2039_4_) {
         super(p_i2039_3_, p_i2039_4_);
         StructureMineshaftPieces.Room room = new StructureMineshaftPieces.Room(0, p_i2039_2_, (p_i2039_3_ << 4) + 2, (p_i2039_4_ << 4) + 2);
         this.components.add(room);
@@ -18,4 +18,4 @@
         this.updateBoundingBox();
         this.markAvailableHeight(worldIn, p_i2039_2_, 10);
     }
-}
+}
